### PR TITLE
Fix #19 and make the login/registration pages nice to look at

### LINKED ISSUE #19: Registration page should be linked on login
There is currently no way to get to the registration page outside of manually entering the URL.

### PR DESCRIPTION
**Pull request by james7132**

### CODE REVIEW

File: src/partials/auth/login.html [hunk 1]
```diff
@@ -1,23 +1,42 @@
...
+      </div>
     </div>
-    <div class="form-group">
-      <label>Password</label>
-      <input type="password" class="form-control" name="password" data-ng-model="loginForm.password" required>
+    <div class="form-group row">
+      <label for="loginPasword" class="col-sm-2 col-form-label">Password</label>
+      <div class="col-sm-10">
+        <input id="loginPasword"
+          type="text"
```
Comment (gscoppino) [inline src/partials/auth/login.html]: Should be of type `password` to hide the value

File: src/partials/auth/register.html [hunk 2]
```diff
@@ -1,25 +1,53 @@
...
+      </div>
     </div>
-    <div class="form-group">
-      <label>Reenter Password</label>
-      <input type="password" class="form-control" name="password" data-ng-model="registerForm.passwordReentry" required />
+    <div class="form-group row">
+      <label for="registerPasswordRentry" class="col-sm-2">Password</label>
+      <div class="col-sm-10">
+        <input id="registePasswordRentry"
+          type="passwordReentry"
```
Comment (gscoppino) [inline src/partials/auth/register.html]: Should be of `type` to `password` to hide the value

File: src/partials/auth/register.html [hunk 3]
```diff
@@ -1,25 +1,53 @@
...
+          required />
+      </div>
     </div>
-    <div class="form-group">
-      <label>Reenter Password</label>
-      <input type="password" class="form-control" name="password" data-ng-model="registerForm.passwordReentry" required />
+    <div class="form-group row">
+      <label for="registerPasswordRentry" class="col-sm-2">Password</label>
+      <div class="col-sm-10">
+        <input id="registePasswordRentry"
```
Comment (gscoppino) [inline src/partials/auth/register.html]: typo in the `id` value (the label right above is referencing the correct value)

File: src/partials/auth/register.html [hunk 4]
```diff
@@ -1,25 +1,53 @@
...
+          required />
+      </div>
     </div>
-    <div class="form-group">
-      <label>Password</label>
-      <input type="password" class="form-control" name="password" data-ng-model="registerForm.password" required />
+    <div class="form-group row">
+      <label for="registerPassword" class="col-sm-2">Password</label>
+      <div class="col-sm-10">
+        <input id="registerUsername"
```
Comment (gscoppino) [inline src/partials/auth/register.html]: typo in the `id` value (the label right above is referencing the correct value)

File: src/partials/auth/login.html [hunk 5]
```diff
@@ -1,23 +1,42 @@
...
+          required>
+      </div>
     </div>
-    <div class="form-group">
-      <label>Password</label>
-      <input type="password" class="form-control" name="password" data-ng-model="loginForm.password" required>
+    <div class="form-group row">
+      <label for="loginPasword" class="col-sm-2 col-form-label">Password</label>
+      <div class="col-sm-10">
+        <input id="loginPasword"
```
Comment (gscoppino) [inline src/partials/auth/login.html]: typo in the `id` value (label right above also has the typo)

**Review by gscoppino — CHANGES_REQUESTED** [review]
- [x] Login/Register look better than before
- [x] Login/Register are suitably responsive to viewport size
- [x] Login/Register are more screen-reader friendly than before
- [x] #19 is fixed

Good stuff @james7132, fix password input types and minor typos then ready to merge

**Comment by james7132** [comment]
PTAL.

**Review by gscoppino — APPROVED** [review]

**Pull request merged**